src/main/org/testng/eclipse/util/CustomSuite.java

```diff
@@ -249,32 +252,47 @@ public File save(File directory) {
     return suiteFile;
   }
 
-  protected void saveSuiteContent(final File file, final XMLStringBuffer content) {
-    FileWriter fw= null;
-    BufferedWriter bw= null;
+  protected void saveSuiteContent(File file, XMLStringBuffer content) {
+    FileOutputStream fileOutputStream = null;
+    BufferedOutputStream bufferedOutputStream = null;
+    OutputStreamWriter osw = null;
     try {
-      fw= new FileWriter(file);
-      bw= new BufferedWriter(fw);
-      bw.write(content.getStringBuffer().toString());
-      bw.flush();
-    }
-    catch(IOException ioe) {
-    }
-    finally {
-      if(null != bw) {
+      fileOutputStream = new FileOutputStream(file);
+      bufferedOutputStream = new BufferedOutputStream(fileOutputStream);
+      osw = new OutputStreamWriter(bufferedOutputStream, Charset.forName("UTF-8"));
+      osw.write(content.getStringBuffer().toString());
+    } catch (IOException localIOException) {
+      if (osw != null)

```

Comment:
You don't have to do this in both the catch block and the finally block. It looks like a duplicate, so you could just delete it all from the catch block and rely on finally being called.